chia/data_layer/data_layer_server.py

```diff
@@ -0,0 +1,64 @@
+import aiosqlite
+from aiohttp import web
+from chia.data_layer.data_store import DataStore
+from chia.util.db_wrapper import DBWrapper
+from chia.types.blockchain_format.tree_hash import bytes32
+from tests.core.data_layer.util import generate_big_datastore
+
+
+class DataLayerServer:
+    async def handle_tree_root(self, request: web.Request) -> web.Response:
```

Comment:
I've wondered if we could use an existing 'server' mechanism instead of our own, but should we continue to use our own system for this as we do for other RPCs/APIs?

Reply:
I've changed it to websockets and passing around JSONs as `aiohttp.WSMsgType.TEXT`.